[2022-03-14 17:59:44,459] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: main_dag.populate_weather_table scheduled__2022-03-14T00:00:00+00:00 [queued]>
[2022-03-14 17:59:44,502] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: main_dag.populate_weather_table scheduled__2022-03-14T00:00:00+00:00 [queued]>
[2022-03-14 17:59:44,503] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-14 17:59:44,505] {taskinstance.py:1244} INFO - Starting attempt 1 of 2
[2022-03-14 17:59:44,506] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-14 17:59:44,560] {taskinstance.py:1264} INFO - Executing <Task(PostgresOperator): populate_weather_table> on 2022-03-14 00:00:00+00:00
[2022-03-14 17:59:44,570] {standard_task_runner.py:52} INFO - Started process 11183 to run task
[2022-03-14 17:59:44,581] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'main_dag', 'populate_weather_table', 'scheduled__2022-03-14T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmpr1u7_3pa', '--error-file', '/tmp/tmp6k5ukh5n']
[2022-03-14 17:59:44,585] {standard_task_runner.py:77} INFO - Job 14: Subtask populate_weather_table
[2022-03-14 17:59:44,715] {logging_mixin.py:109} INFO - Running <TaskInstance: main_dag.populate_weather_table scheduled__2022-03-14T00:00:00+00:00 [running]> on host 0bb54ecb374d
[2022-03-14 17:59:44,859] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=main_dag
AIRFLOW_CTX_TASK_ID=populate_weather_table
AIRFLOW_CTX_EXECUTION_DATE=2022-03-14T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-14T00:00:00+00:00
[2022-03-14 17:59:44,884] {base.py:79} INFO - Using connection to: id: postgres_default. Host: postgres, Port: 5432, Schema: postgres, Login: ***, Password: ***, extra: {}
[2022-03-14 17:59:44,899] {dbapi.py:225} INFO - Running statement: DROP TABLE IF EXISTS weather_table;
CREATE TABLE weather_table (
state_name VARCHAR NOT NULL,
description VARCHAR NOT NULL,
temperature VARCHAR NOT NULL,
feels_like VARCHAR NOT NULL,
temp_min VARCHAR NOT NULL,
temp_max VARCHAR NOT NULL,
humidity VARCHAR NOT NULL,
clouds VARCHAR NOT NULL);, parameters: None
[2022-03-14 17:59:44,939] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=main_dag, task_id=populate_weather_table, execution_date=20220314T000000, start_date=20220314T175944, end_date=20220314T175944
[2022-03-14 17:59:45,005] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-14 17:59:45,104] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
